src/cartservice/cartservice.csproj

```diff
@@ -14,7 +14,11 @@
     <PackageReference Include="grpc.tools" Version="1.12.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.1.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.1.1" />
-    <PackageReference Include="StackExchange.Redis" Version="2.0.601" />
+    <PackageReference Include="StackExchange.Redis" Version="2.1.58" />
+    <PackageReference Include="xunit" Version="2.4.1" />
```

Comment:
xunit got added?

Reply:
There is no vs solution file and when I create one it won't build locally without the xunit package. It will still build in the docker image. I could include my solution file or remove this, any thoughts?